### DISCUSSION TITLE
underfull问题

### DISCUSSION THREAD

**Discussion by ylxiang2018** [sjtug/SJTUThesis#716] [提问]
使用最新版的模板导入overleaf，编译出现警告：
Underfull \hbox (badness 2269) in paragraph at lines 59--59

请问，是什么原因？怎么解决呢？

**Comment by chenhongyang113** [comment]
同问这个问题，用的vscode，编译的时候提示这个警告，这个问题是什么原因造成的呢？搜了一圈没有很理解

**Comment by LogCreative** [comment]
这个是 info，只是说有一些部分没有填满水平盒限定范围，基本不影响正常排版。

**Reply by LogCreative** [reply]
对于每一行排版，一般都有一个额定宽度。由于比如插入了内联代码块等等原因，根据 (La)TeX 的断行算法，可能会导致某行的字符排版总宽达不到这个宽度，根据行文设置的不同，可能会采取增大字符间距的方式填满该行（两端对齐），比如
<img width="344" alt="image" src="https://user-images.githubusercontent.com/61653082/158004131-7f37f02d-a227-4879-af9e-047ac0a5f04a.png">
这种情况可以考虑对这一部分多加一些字，让该行看起来没那么稀疏。当然如果是不可以更改的内容（比如 59 是引用部分的内容），直接忽略就可以。

**Reply by chenhongyang113** [reply]
谢谢解答，明白了，我刚刚遇到的问题就是在一行的末尾原文是“（A-B）”，就会报错，但是我前面加几个字把（A-B）全挤到下一行就每天警告了，结合您的解答，大概理解这个警告是什么意思了

**Reply by LogCreative** [reply]
嗯 实际上这个算不上警告（warning），只是最低优先级的提示（info）而已
